stackslib/src/chainstate/coordinator/mod.rs

```diff
@@ -3052,6 +3053,7 @@ impl<
                                     return Ok(Some(pox_anchor));
                                 }
                             }
+                            StacksEpochId::Epoch30 => todo!(), // Need input here
```

Comment:
This should have the same behavior as Epochs 2.1 and 2.4 (for now: this will likely be changed in nakamoto, but in this PR, it should just share that behavior with 2.1-2.4).

Reply:
Okay, moved into their match arm